src/lava/magma/core/process/process.py

```diff
@@ -249,6 +249,12 @@ def __init__(self, **kwargs):
         # Current runtime environment
         self._runtime: ty.Optional[Runtime] = None
 
+    def __del__(self):
+        """On destructor call automatically
```

Comment:
Alternative docstring suggestion: "On destruction, terminate Runtime automatically to free compute resources."